src/main/kotlin/com/amazon/opendistroforelasticsearch/indexstatemanagement/resthandler/RestRetryFailedManagedIndexAction.kt

```diff
@@ -64,7 +64,7 @@ class RestRetryFailedManagedIndexAction(
         return "retry_failed_managed_index"
     }
 
-    @Suppress("SpreadOperator")
+    @Suppress("SpreadOperator") // There is no way around dealing with java vararg without spread operator.
```

Comment:
I think it is good to have this so that we don't accidentally use "SpreadOperator". Again only reason we are using is because we need to pass into java `vararg` which we have no other option.

Reply:
Sure, do you want to move it to the line directly above its use then? Otherwise this suppresses the whole method which could theoretically suppress a use of it that we do not want to.

Reply:
As discussed offline, it distracts the flow of the code by adding the `Suppress` directly above the code. I will keep the way it is.